libcontainer/nsenter/nsexec.c

```diff
@@ -24,21 +22,21 @@
 
 /* All arguments should be above stack, because it grows down */
 struct clone_arg {
-	/*
-	 * Reserve some space for clone() to locate arguments
-	 * and retcode in this place
-	 */
-	char stack[4096] __attribute__ ((aligned(16)));
-	char stack_ptr[0];
-	jmp_buf *env;
+    /*
+     * Reserve some space for clone() to locate arguments
+     * and retcode in this place
+     */
+    char stack[4096] __attribute__ ((aligned(16)));
+    char stack_ptr[0];
+    jmp_buf *env;
```

Comment:
mmm, what i did was to use indent -linux with https://github.com/crosbymichael/vim-cfmt